staging/src/k8s.io/client-go/util/flowcontrol/backoff.go

```diff
@@ -100,7 +100,10 @@ func (p *Backoff) Next(id string, eventTime time.Time) {
 	} else {
 		delay := entry.backoff * 2       // exponential
 		delay += p.jitter(entry.backoff) // add some jitter to the delay
-		entry.backoff = time.Duration(integer.Int64Min(int64(delay), int64(p.maxDuration)))
+		// instead of returning a fixed value of p.maxDuration, we can
+		// jitter the maximum duration.
+		maxDurationJittered := p.maxDuration + p.jitter(p.maxDuration)
+		entry.backoff = time.Duration(integer.Int64Min(int64(delay), int64(maxDurationJittered)))
```

Comment:
why do we need to jittered values, we already add jitter to the delay

Reply:
also, this is breaking

> // move backoff to the next mark, capping at maxDuration

no?

Reply:
I removed it, i think the preceding sequence of jittered delays will prevent a synchronized cluster of calls on the server.